### PR TITLE
#160583965 Users should be able to comment on articles

### PR DESCRIPTION
**Pull request by j0flintking02**
#### What does this PR do?
Users should be able to comment on articles
#### Description of Task to be completed?
Comments allow authors to get feedback on their articles.
We all love to know what people think about what we put out to the world
#### How should this be manually tested?
- clone the project
```console
git clone https://github.com/andela/ah-frontend-stark.git
```
- move into the project folder
```console 
cd ah-frontend-stark
```
- install the dependency
``` console 
yarn install
```
- run the project
```console
yarn start
```
navigate to one of the comment and there should be a comment box
```
url /article/some-other-stuff-to-talk-about12345
```
#### What are the relevant pivotal tracker stories?
[#160583965](https://www.pivotaltracker.com/n/projects/2198584/stories/160583965)
#### Screenshots (if appropriate)
screen shot 2018-11-12 at 15 19 46

### CODE REVIEW

**Review by balaakagordon — APPROVED** [review]
LGTM

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,5 @@ coverage
 
 # DS store
 .DS_Store
+
+.idea/*
```
Comment (lym) [inline .gitignore]: Add a new line here

File: src/App.js [hunk 2]
```diff
@@ -14,7 +14,6 @@ import Register from './views/Register';
 import TempHome from './components/TempHome';
 
 
-
```
Comment (lym) [inline src/App.js]: Remove this new blank line. Let's have only two empty lines between import and class declaration.

File: src/actions/commentAction.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+export const fetchAction = slug => (dispatch) => {
+  fetch(`https://ah-backend-stark-staging.herokuapp.com/api/articles/${slug}/comments/`)
+    .then(response => response.json())
+    .then((data) => {
+      dispatch({
+        type: GET_COMMENT,
+        payload: data.comments,
+      });
+    }).catch(error => console.error('Fetch Error =\n', error));
+};
```
Comment (lym) [inline src/actions/commentAction.js:17]: Add a blank line here

File: src/tests/commentReducer.test.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      comment: {
+        body: 'hey there',
+      },
+    };
+    const response = { comments: [{ comment: { body: 'hey there' } }] };
+    expect(commentReducer(initialState, {
+      type: actions.ADD_COMMENT,
+      payload: newComment,
+    })).toEqual(response);
+  });
```
Comment (lym) [inline src/tests/commentReducer.test.js:18]: Have at least a blank line separating the examples, for easy readability.

File: src/tests/comments.test.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+      <Provider store={store}>
+        <Comment fetchAction={jest.fn} slug="how-i-meet-your-mothersssss" />
+      </Provider>,
+    );
+  });
+  console.log(wrapper);
+  it('should render comments comment', () => {
+    expect(wrapper).toHaveLength(1);
+  });
+});
```
Comment (lym) [inline src/tests/comments.test.js:39]: Add a line here for easy readability.

**Review by lym — CHANGES_REQUESTED** [review]

**Review by kenneth051 — CHANGES_REQUESTED** [review]
Good job Jonathan, Could you please make the PR more descriptive. I don't know what URL link to go to when testing the feature. Thank you

**Review by kenneth051 — APPROVED** [review]

**Pull request merged**